code/datums/gamemode/factions/bloodcult/bloodcult_runes.dm

```diff
@@ -494,4 +494,61 @@ var/list/uristrune_cache = list()//icon cache, so the whole blending process is
 			if (src && loc)
 				conceal_cooldown = 0
 		return 1
-	return 0
\ No newline at end of file
+	return 
+
+// -- Warding runes, detect ennemies
+/obj/effect/rune/ward/
+	var/uses = 5
+	var/last_threeshold = -1
+
+/obj/effect/rune/ward/New()
+	conceal()
+	return ..()
+
+/obj/effect/rune/ward/Crossed(var/atom/movable/mover)
+	// Triggers if a living non-cultist mob crosses the rune.
+	if (!isliving(mover))
+		return
+	var/mob/living/L = mover
+	if (uses < 0)
```

Comment:
This makes it 6 uses because it has 5 and it only disappears if it's lower than 0, not if it's at 0 or lower

Reply:
It is also only checked if it's crossed so that means if it reaches at -1 and someone crosses it THEN it deletes

Reply:
The second point is not really that big of an issue, I think.